pex/pip/tool.py

```diff
@@ -742,16 +698,3 @@ def spawn_debug(
         return self._spawn_pip_isolated_job(
             debug_command, pip_verbosity=1, stdout=subprocess.PIPE, stderr=subprocess.PIPE
         )
-
-
-_PIP = {}  # type: Dict[Optional[PythonInterpreter], Pip]
-
-
-def get_pip(interpreter=None):
```

Comment:
And this went here: https://github.com/pantsbuild/pex/pull/1892/files#diff-a6a6ecdb319224fc07ed4e7a1a69686efc58ce1501e65eb057e684e907f34a12R192